### ISSUE TITLE
Allow syncing without defining projects

### ISSUE THREAD

**Issue by laggingreflex** [appnexus/sicksync#78]
Sometimes it's too restrictive that I cannot start syncing any current folder.

How about if it could just let you sync current dir by just asking you which path on server you want to sync to. 

Like this:
```
sicksync --here --server <server-name> --path /some/path/on/server 
```

**Comment by joelgriffith** [comment]
This is a great idea, how about:

`sicksync quicksync --server <user@remotehost.edu:/some/path/on/server >`

This matches the semantics of `scp`, which is pretty standardized. Under-the-hood it could execute an rsync, but I don't think it could watch wo doing some sort of installation on the remote machine... I'd be open to ideas for that

**Comment by laggingreflex** [comment]
SOunds good.
 
>wo doing some sort of installation

Do you mean installation of sicksync itself, or some initialization stuff that sicksync does when it "creates" a new project? I already do have sicksync on the server. Would the `quicksync` command (locally) not be enough to tell the server to "get ready to receive files"? Because in my other request #79 I actually wanted avoid rsync. Do you mean that rsync is *all* that'll be possible here with quicksync?

Or did you mean that if one wanted to avoid installation of sicksync on the server then in that case rsyc would be all that could be done? Installation of sicksync on the server isn't a problem for me personally.

BTW this got me thinking about why is sicksync required to be installed on server in the first place? I used to use [WinSCP](https://winscp.net) which didn't require it to be installed on server.

**Comment by joelgriffith** [comment]
> Do you mean installation of sicksync itself

Yes, I was envisioning you'd run this on a server that doesn't have sicksync already installed. If it _does_, then this command could work entirely like `sicksync start` then. So if if it's not an issue, then it'd be fine.

> BTW this got me thinking about why is sicksync required to be installed on server in the first place? I used to use WinSCP which didn't require it to be installed on server.

So, if `scp` or some variant is sufficient, then you probably don't need sicksync at all. My use case was: I had a VPM in NYC, but my machine I sync to was in LA, which meant each file-change had to go to the east coast and back. Since websockets persist, in a sense, my file-changes went from seconds down to near-instantaneous speed.

I'll whip together a `quicksync` command for the next version. Thanks!
